api_spec.yaml

```diff
@@ -1752,6 +1760,81 @@ paths:
                                 type: string
                                 enum:
                                     - Unauthorised access
+    /dashboard/api/tenants/third-party/boxy-saml:
+        put:
+            tags:
+                - Multitenancy
+            summary: Create or update third party config for a tenant with BoxyHQ
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                thirdPartyIdSuffix:
+                                    type: string
+                                boxyUrl:
+                                    type: string
+                                boxyApiKey:
+                                    type: string
+                                encodedRawMetadata:
+                                    type: string
```

Comment:
Should we also provide an option to specify SAML URL? Generally devs have a link to their SAML, if they provide that, we can download in the backend. Please check if there is a possibility.

Reply:
Yes that makes sense, we can use that URL to fetch the XML on the server and then encode it to Base64, have added another optional field for it